cmake/python.cmake

```diff
@@ -57,7 +57,11 @@ endif()
 
 # Use the Python module to find the python lib.
 if(BUILD_WITH_PYTHON_3)
-    find_package(Python 3.7 EXACT REQUIRED COMPONENTS Interpreter)
+    # We support two versions of python 3 (3.7/3.9).
+    find_package(Python 3.9 EXACT QUIET COMPONENTS Interpreter)
+    if (NOT Python_FOUND)
+        find_package(Python 3.7 EXACT REQUIRED COMPONENTS Interpreter)
+    endif()
```

Comment:
Same as above, but this code is only used when no python (via vars) was sent in. Called from CMakeLists.txt (above) line 95.